rts/Rendering/Fonts/CFontTexture.cpp

```diff
@@ -461,7 +460,7 @@ static std::shared_ptr<FontFace> GetFontForCharacters(const std::vector<char32_t
 
 		if (family)
 			FcPatternAddString(pattern, FC_FAMILY, family);
-		if (foundry)
+		if (foundry && strcmp("UKWN", reinterpret_cast<char*>(foundry)) != 0 && strcmp("ukwn", reinterpret_cast<char*>(foundry)) != 0)
```

Comment:
Changed that.

Regarding UKWN and ukwn case sensitivity, I found this seeming to imply just UKWN and ukwn are possible https://github.com/fonttools/fontbakery/issues/713, also https://github.com/fonttools/fontbakery/blob/c9954809500b6a2fe4b03995594b917167315ecc/Lib/fontbakery/checks/vendorspecific/googlefonts/os2.py#L104 has some other bad codes but not sure we should filter by those too since we're not having any problems related to that and seems complicating the code for nothing atm.